app/assets/javascripts/admin/auto-accept-cookies.js

```diff
@@ -0,0 +1,5 @@
+window.GOVUK = window.GOVUK || {};
+(function (GOVUK) {
+  'use strict'
+  GOVUK.setDefaultConsentCookie()
+}(window.GOVUK))
```

Comment:
A simpler way of doing this (which I think should work, haven't tested but [seen implemented successfully elsewhere](https://github.com/alphagov/content-tagger/pull/1657/files)) would be to add the following at the end of `application.js`. The analytics code is initialised in a document.ready so it's fine to do it like this. Also I think you also need the second line.

```
window.GOVUK.approveAllCookieTypes()
window.GOVUK.cookie('cookies_preferences_set', 'true', { days: 365 })
```